pyproject.toml

```diff
@@ -39,6 +39,7 @@ dependencies = [
     "litdata>=0.2.49",
     "seaborn>=0.13.2",
     "dgeb>=0.2.0",
+    "rdkit>=2025.3.3",
```

Comment:
thanks! this is actually included in the confusingly named `mgm` (an internally used name) optional dependencies set: https://github.com/prescient-design/lobster/blob/main/pyproject.toml#L64

so this can be deleted.